### ISSUE TITLE
Flairs support

### ISSUE THREAD

**Issue by Lavko** [JedS6391/Reddit.NET#16]
Hey, I looked for Flair in the code but didn't find anything. Is there any way to get and send flairs with submission?

**Comment by JedS6391** [comment]
Hi there,

It's not currently supported but I've started on adding the ability to get/set submission flairs in #17 if you're interested in trying it out. Still need to implement setting flairs for a submission, but get is mostly working.


```csharp
var askScience = client.Subreddit("askscience");

// Get available flairs for submissions in this subreddit
var flairs = await askScience.GetSubmissionFlairsAsync();

var submissions = askScience.GetSubmissionsAsync(builder =>
    builder
        .WithSort(SubredditSubmissionSort.Hot)
        .WithMaximumItems(10));

await foreach (var submission in submissions)
{
    // Display flair for submission
    Console.WriteLine($"{submission.Title} [{submission.FlairText}]");
}
```

**Comment by JedS6391** [comment]
Hi again, support for flairs is now available in [version 1.3.0](https://github.com/JedS6391/Reddit.NET/blob/master/CHANGELOG.md#130---2022-10-28).

**Comment by Lavko** [comment]
Great work! Thank you.
